xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -575,7 +563,6 @@ bool CDVDVideoCodecFFmpeg::AddData(const DemuxPacket &packet)
     m_started = true;
 
   m_dts = packet.dts;
-  m_pCodecContext->reordered_opaque = pts_dtoi(packet.pts);
```

Comment:
If you put that in again - does tvheadend work again?

Reply:
no, it doesnt